proto/def.bzl

```diff
@@ -38,20 +38,27 @@ load(
 GoProtoImports = provider()
 
 def get_imports(attr):
-    direct = []
+    protos = []
+
     if hasattr(attr, "proto") and hasattr(attr.proto, "proto"):
-        proto = attr.proto.proto
-        direct = [
-            "{}={}".format(proto_path(src, proto), attr.importpath)
-            for src in proto.check_deps_sources
-        ]
+        protos = [attr.proto.proto]
+    elif hasattr(attr, "protos"):
+        for proto in attr.protos:
+            if hasattr(proto, "proto"):
+                protos.append(proto.proto)
+
+    direct = dict()
+    for proto in protos:
+        for src in proto.check_deps_sources:
```

Comment:
I think `.to_list()` is needed here (previous code was incorrect, too). Maybe run tests with `--incompatible_depset_is_not_iterable` to double-check there aren't any other issues.

Reply:
changed